src/lib/renderer-types.ts

```diff
@@ -47,6 +47,15 @@ export interface Renderer {
    */
   canvas: HTMLCanvasElement;
 
+  /**
+   * After the canvas changes shape, this method instructs the Scene to adjust
+   * the offset and canvas properties (width, height) to accommodate.
+   *
+   * @param fixedWorldPoint Optional world point to preserve relative to the
```

Comment:
Not sure this makes sense, but perhaps this should be a pixel space coordinate, e.g. the center pixel? It looks like e.g. Google Maps keeps whatever was in the center of the screen in the center after resize.

Reply:
That sounds reasonable.

Reply:
Done. Changed to fixedCanvasPoint, defaults to center.